kover-maven-plugin/docs/index.md

```diff
@@ -0,0 +1,333 @@
+
+In this case, it becomes necessary to generate reports for several modules at once.
+
+In order to activate the creation of reports on several projects, you need to add the following configuration
+`<aggregate>true</aggregate>` into the Kover Plugin `configuration` block.
+
+Then, the classes in the report will be collected for all modules specified in the `dependencies` block, except for those for which the scope is `test`.
+And the coverage will be collected from tests located in modules that are also in the `dependencies` block, including dependencies with the `test` scope.
+
+See full [example project](https://github.com/Kotlin/kotlinx-kover/tree/main/kover-maven-plugin/examples/merged-report)
+
+## Configuration
+
+All available configuration options are shown below:
+```xml
+
+<plugin>
+    <groupId>org.jetbrains.kotlinx</groupId>
+    <artifactId>kover-maven-plugin</artifactId>
+    <version>0.8.2</version>
+    <executions>
+        <!-- instrument test tasks -->
+        <execution>
+            <id>instr</id>
+            <goals>
+                <goal>instrumentation</goal>
+            </goals>
+        </execution>
+        
+        <!-- generate XML report in verify phase -->
+        <execution>
+            <id>kover-xml</id>
+            <goals>
+                <goal>report-xml</goal>
+            </goals>
+        </execution>
+        
+        <!-- generate HTML report in verify phase -->
+        <execution>
+            <id>kover-html</id>
+            <goals>
+                <goal>report-html</goal>
+            </goals>
+        </execution>
+
+        <!-- check coverage rules in verify phase -->
+        <execution>
+            <id>kover-verify</id>
+            <goals>
+                <goal>verify</goal>
+            </goals>
+        </execution>
+
+        <!-- generate IC report -->
+        <execution>
+            <id>kover-ic</id>
+            <goals>
+                <goal>report-ic</goal>
+            </goals>
+        </execution>
+
+        <!-- print coverage values to the log -->
+        <execution>
+            <id>kover-log</id>
+            <goals>
+                <goal>log</goal>
+            </goals>
+        </execution>
+    </executions>
+
+    
+    <configuration>
+        <!-- Skip execution of Kover goals -->
+        <!-- Boolean, default: false -->
+        <skip>false</skip>
+        
+        <!-- Classes excluded from instrumentation. -->
+        <!-- The specified classes will not be modified when loaded into the JVM and the coverage for them will always be 0%. -->
+        <!-- Wildcards `*` and `?` are acceptable. -->
+        <!-- List of String values, default: empty list -->
+        <uninstrumentedClasses>com.example.*,excluded.from.instrumentation.*</uninstrumentedClasses>
+
+        <!-- Property name to pass java agent argument to the JVM in which tests are running. -->
+        <!-- String, default: argLine -->
+        <agentPropertyName>argLine</agentPropertyName>
```

Comment:
Because by default, the maven surefire plugin reads the value of the `argLine` property and substitutes it as arguments to the JVM on which the tests are run